example/accel/offload-template/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+#------------------------------- -*- cmake -*- -------------------------------#
+# Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+#-----------------------------------------------------------------------------#
+# Minimal Geant4 application with Celeritas offloading capabilities
+#-----------------------------------------------------------------------------#
+cmake_minimum_required(VERSION 3.10)
+cmake_policy(VERSION 3.12...3.30)
+project(offload-template)
+
+# Export/set Celeritas_ROOT environment variable pointing to its install dir
+# Defaults to celeritas/install
+set(ENV{Celeritas_ROOT} "../../../../install")
+
+#-----------------------------------------------------------------------------#
+# Find packages
+find_package(Geant4 REQUIRED)
+find_package(Celeritas 0.5 REQUIRED)
```

Comment:
Since Celeritas points to Geant4 but not vice versa, switch these (otherwise the user will have to set both paths).